### DISCUSSION TITLE
Custom badge caption

### DISCUSSION THREAD

**Discussion by YounesCheikh** [danielpalme/ReportGenerator#493] [Q&A] [answered]
Hello, 
I'm using report generator to generate badge for test coverage to display it on Github repository (Readme). 
My repository has 2 branches (by the moment) and I want to display test coverage for each branch. Currently I have only one with caption `Coverage`
![one badge](https://raw.githubusercontent.com/wiki/YounesCheikh/Cliargs.NET/combined.svg). 

**My question:** 
Is there a way to customise the badge caption in order to get a generated with caption "`main test coverage`" or "`release test coverage`" ? 

Thanks in advance.

**Comment by danielpalme** [comment, marked as answer]
There is no way to customize the caption.
But if you add a little Powershell command to your build pipeline, you can change the caption yourself:

```
(gc "badge_combined.svg") | % { $_ -replace "Coverage", "main" } | Out-File "badge_combined.svg" -Encoding UTF8
```
